modules/nodes-nodes-kernel-arguments.adoc

```diff
@@ -136,11 +136,16 @@ Starting pod/ip-10-0-141-105ec2internal-debug ...
 To use host binaries, run `chroot /host`
 
 sh-4.2# cat /host/proc/cmdline
-BOOT_IMAGE=/ostree/rhcos-... console=tty0 console=ttyS0,115200n8
+BOOT_IMAGE=/ostree/rhcos-... ,115200n8
```

Comment:
I don't think this needs to be changed; these are baked in default values in our images